README.md

```diff
@@ -1,6 +1,6 @@
 # OpenStreetMapSpeeds Schema
 
-This repository houses both a description and outline of the JSON schema for the data artifact the conflation repository produces as well as the latest version of the artifact itself.
+This repository houses both a description/outline of the JSON schema for the data artifact the [conflation](https://github.com/OpenStreetMapSpeeds/conflation) repository produces as well as the latest version of the artifact itself.
```

Comment:
nit: "houses both a description and outline" or "houses a description/outline"

Reply:
the "both" refers to the schema and that data. ill reword this clunky nonsense, sorry